src/models/pipeline-run-task/test-data.ts

```diff
@@ -0,0 +1,61 @@
+import { PipelineRunTaskModel } from '.';
+
+import { TestDependencyDataType } from '../model-types';
+import { ProgressStatus } from '../progress-status';
+
+/*
+ * TestData array
+ *
+ * Note: First object should always be valid
+ */
+export const testData: TestDependencyDataType<PipelineRunTaskModel> = [
+  {
+    __testDescription: 'Valid full object',
+    name: 'some-task',
+    realName: 'tkn-task-dev-abcde-some-zxcv-20220510010101',
+    pipelineRunEnv: 'dev',
+    pipelineName: 'build-pipeline',
+    started: new Date('2022-05-10T14:31:23Z'),
+    ended: new Date(),
+    status: ProgressStatus.Succeeded,
+  },
+  {
+    __testDescription: 'Valid partial object',
+    name: 'some-task',
+    realName: 'tkn-task-dev-abcde-some-zxcv-20220510010101',
+    pipelineRunEnv: 'dev',
+    pipelineName: 'build-pipeline',
+    started: new Date('2022-05-10T14:31:23Z'),
+    status: ProgressStatus.Queued,
+  },
+  {
+    __testDescription: 'Invalid full object',
+    __testIsInvalidSample: true,
+    name: 'some-task',
+    realName: '',
+    pipelineRunEnv: 'dev',
+    pipelineName: 'build-pipeline',
+    started: new Date('2022-05-10T14:31:23Z'),
+    ended: new Date(),
+    status: ProgressStatus.Succeeded,
```

Comment:
add missing optional object in 'Invalid full object' test
```suggestion
    status: ProgressStatus.Succeeded,
    statusMessage: 'statusMessage',
```